docs/mp/health/01_introduction.adoc

```diff
@@ -26,68 +26,92 @@
 :common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
 :microprofile-bundle: true
 
-include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+== ToC
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Reference, Reference>>
+- <<Examples, Examples>>
```

Comment:
There are multiple sections in examples, I think they deserve their own place in TOC - especially so we can share links pointing directly to them

Reply:
I don't think it's a good idea to overload ToC. It will be displayed in a floating panel at the right side in the next version of docs. Anchors for every sample is present, so links to particular samples can be shared.